docs/data/toolpad/core/components/use-notifications/use-notifications.md

```diff
@@ -0,0 +1,80 @@
+---
+productId: toolpad-core
+title: useNotifications
+components: NotificationsProvider
+---
+
+# Notifications
+
+<p class="description">Imperative APIs to show and interact with application notifications</p>
+
+Toolpad core offers a set of abstractions that make it easier to interact with notifications. Notifications are used to give short updates to the user about things that are happening during the application lifetime. They appear at the bottom of the screen. The Toolpad API allows for opening multiple notifications concurrenlty.
+
+First thing you need to do to get access to the notifications APIs is install the NotificationsProvider.
+
+```js
+import { NotificationsProvider } from '@toolpad/core/useNotifications';
+
+function App({ children }) {
+  return <NotificationsProvider>{children}</NotificationsProvider>;
+}
+```
+
+Now you can get acess to the notifications APIs through the `useNotifications` hook.
+
+```js
+import { useNotifications } from '@toolpad/core/useNotifications';
+
+function MyApp() {
+  const notifications = useNotifications();
+  // ...
+}
+```
+
+:::info
+The Toolpad AppProvider automatically comes with notifications installed. You won't need to explicitly add the NotificationsProvider in Toolpad applications.
+:::
+
+## Basic notification
+
+You can notify your users with a neutral message by calling `notifications.show`. To have the notification automatically hide, add the `autoHideDuration` option. This expresses the time in milliseconds after which to close the notification.
+
+{{"demo": "BasicNotification.js"}}
+
+## Alert notifications
+
+You can send notifications under the form of alerts with the `severity` property. It takes a value from `"info"`, `"success"`, `"warning"`, or `"error"`.
+
+{{"demo": "AlertNotification.js"}}
+
+## Multiple notifications
```

Comment:
Maybe we could stack them vertically, but that could take some work to do well so could just be a later improvement.

Reply:
There is a section that talks about "notification center". a way to view and manage all open notifications at once.